PolyEngine/CMakeLists.txt

```diff
@@ -5,6 +5,10 @@ project(PolyEngine VERSION 0.1.2 LANGUAGES CXX C) #note(vuko): without C older C
 # Initial setup: C++ standard version, build types, CMake modules
 ##
 
+if (${CMAKE_SYSTEM_NAME} MATCHES "Darwin")
+    set(MACOSX TRUE)
```

Comment:
https://cmake.org/cmake/help/latest/variable/APPLE.html 😛 

Well, unless you want to cross-compile. But we don't support that. (yet?)